common/lib/common-definitions/src/fluidError.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryProperties } from "./logger";
+
+ /**
+ * Can be decl merged to add stuff like packagename, odspErrorResponse, etc
+ * The logger impl can choose to log anything in here as appropriate, but by default it's not
+ */
+export interface ISensitiveDebugData {
```

Comment:
Shouldn't it allow more granularity?
What is sensitive will be determined by environment where code is run. So I think the code that issues telemetry events should classify content (for example "contains component ID" or "contains error from 3rd party code", etc.) and the hosting layer (the one implementing telemetry interface) will determine if it's appropriate or not to allow that info through.

Reply:
For example below, stack might be sensitive or not (i.e. it may contains only data from our stack, or maybe not).
I assume here that users saving our app locally and running it from local disk are violating terms of service and we could collect it in such case (worth confirming).